addons/point_of_sale/models/pos_order.py

```diff
@@ -616,17 +616,24 @@ def refund(self):
 
     @api.model
     def action_receipt_to_customer(self, name, client, ticket, order_ids=False):
-        template_obj = self.env['mail.mail']
-        message = "<p>Dear %s,<br/>Here is your electronic ticket from the %s. </p>" % (client['name'], name)
+        orders = self.browse(order_ids) if order_ids else self
+        if not orders:
```

Comment:
So I've just checked, and order_ids seems to be always sent to the function (before it was only when invoiced, but now it's always). So I can change it in multi for master, and maybe just check the group here.

I'll change it in master once this one is forwardported.